provision_host.sh

```diff
@@ -22,7 +22,13 @@ if echo "${IMAGE_NAME}" | grep -qi centos 2>/dev/null ; then
 else
     OS_TYPE=unknown
 fi
-./scripts/v1alpha1/user_data.sh "${BMHOST}" ${OS_TYPE} | kubectl apply -n metal3 -f -
+
+if [ "${CAPI_VERSION}" == "v1alpha3" ]; then
+  ./scripts/v1alpha1/user_data.sh "${BMHOST}" ${OS_TYPE} | kubectl apply -f -
+else
+  ./scripts/v1alpha1/user_data.sh "${BMHOST}" ${OS_TYPE} | kubectl apply -n metal3 -f -
+fi
```

Comment:
the bmh are still created in metal3, no ?

Reply:
yes, correct